convert2rhel/logger.py

```diff
@@ -176,6 +176,10 @@ class bcolors:
     ENDC = "\033[0m"
 
 
+def colorize(message, color="OKGREEN"):
```

Comment:
no docstrings here, not sure if you want to add one, @abadger

Reply:
Now with docstring